src/frontends/pytorch/src/op/hardtanh.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
```

Comment:
should we include specific operation instead of heavy common header?

Reply:
Good question, that is how it is done in other frontends, like tf or paddle, I remember it was considered a better approach then specific operation, but I don't remember strong reasons and I don't think we thought about compile time before.